README.md

```diff
@@ -1,75 +1,75 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone project: Entrepreneurs Forum Summit landing page
 
 > Description the project.
```

Comment:
- [ ] Please as requested by the previous reviewer, kindly add a brief description of what this project is all about.